app/src/main/java/com/example/universitieslist/view/components/UniversityList.kt

```diff
@@ -5,10 +5,13 @@ import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import com.example.universitieslist.data.model.UniversityResponse
 import com.example.universitieslist.ui.theme.Dimensions.dimens20
 
 @Composable
-fun UniversityList() {
+fun UniversityList(
+    universityList: List<UniversityResponse>,
```

Comment:
You should not pass your data layer data classes in the ViewModel particularly when it is an entity class. Rather you should have a data class for the view which should have been mapped in the repository layer.

For example, you should have a function that maps `UniversityResponse` to `University` in the `University` class.

Reply:
I passed it since the data class is not an entity class but just a `Moshi Json` serialisation class.
There's no `ROOM` functionality on the project yet.

Reply:
I think you should still separate them. What if you need to implement DB later?

Reply:
okay, will do